modules/specifying-openshift-installer-log-levels.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-installations.adoc
+
+[id="specifying-openshift-installer-log-levels_{context}"]
+= Specifying Openshift installer log levels
+
+By default, the {product-title} installer log level is set to `info`. If more detailed logging is required when diagnosing a failed {product-title} {product-version} installation, increase the `openshift-install` log level to `debug` when starting the installation again.
+
+.Prerequisites
+
+* Access to the installation host where `openshift-install` is being run.
+
+.Procedure
+
+. Set the installation log level to `debug` when initiating the installation:
```

Comment:
If a procedure only has one step, then it should be an unordered bullet.

Reply:
Ok thank you. Resolved.